draft-ietf-ppm-dap.md

```diff
@@ -1675,27 +1661,19 @@ and Helper then run the aggregate initialization flow to accomplish two tasks:
 
 The Leader and Helper initialization behavior is detailed below.
 
-Implementation note: the Leader will generally want to associate each report
-with a single aggregation job, as otherwise the duplicated reports will
-eventually be discarded as a replay. However, it is likely not appropriate to
-directly use the used-ID storage used for replay protection to determine which
-reports can be added to an aggregation job: certain errors (e.g.
-`report_too_early`) allow the report to be added to another aggregation job in
-the future; but storage into the used-ID storage is permanent.
```

Comment:
brevity: this seems specific to a particular implementation and doesn't fit in the specification.

Reply:
SGTM.